tests/vmi_multus_test.go

```diff
@@ -863,6 +865,92 @@ var _ = Describe("[Serial]SRIOV", func() {
 			Expect(libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB))).To(Succeed())
 			Expect(libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))).To(Succeed())
 		})
+
+		Context("VLANed SR-IOV network with a vlaned VMI", func() {
+			var (
+				vlanedVMI1 *v1.VirtualMachineInstance
+				macVlaned1 string
+			)
+
+			const (
+				cidrVlaned1          = "192.168.0.1/24"
+				sriovVlanNetworkName = "sriov-vlan"
+			)
+
+			generateRandomMacAsString := func() string {
+				mac, err := tests.GenerateRandomMac()
+				Expect(err).ToNot(HaveOccurred())
+				return mac.String()
+			}
+
+			BeforeEach(func() {
+				macVlaned1 = generateRandomMacAsString()
+				createNetwork(sriovVlanNetworkName, tests.NamespaceTestDefault, sriovConfVlanCRD)
+				vlanedVMI1 = createSriovVMIIWithStaticIPByMac(sriovVlanNetworkName, macVlaned1, cidrVlaned1)
+			})
+
+			AfterEach(func() {
```

Comment:
This pattern has been freezed for now, as such a deletion will remove the objects and we will not collect them in the log collection when a test fails.

We need a "clean all" at the beginning.